app/src/main/java/com/brewfind/ui/main/MainViewModel.kt

```diff
@@ -16,4 +16,8 @@ class MainViewModel : ViewModel() {
     fun fetchBreweries() {
         breweries = breweryService.fetchBreweries()
     }
+
+    fun fetchBreweriesByName(breweryName: String){
+        breweries = breweryService.fetchBreweriesByName(breweryName)
+    }
```

Comment:
Added a new method which would call the new method in breweryService, which would then store the data into a variable as a string.